web/src/components/Both.js

```diff
@@ -1,11 +1,13 @@
 import React, { Component } from "react";
 import { SketchPicker } from 'react-color';
+// CR * formatting
```

Comment:
Most of these component files NEED to be reformatted. The JSX is pretty challenging to visually parse. Consider installing `prettier` for VSCode and adding a `.prettierconfig` to your codebase. Then it's just a `Alt+Shift+F` away! 🕺 🤸‍♀️